requirements-docs.txt

```diff
@@ -1,12 +1,13 @@
+ablog<0.11
+matplotlib
 myst-nb
-sphinx>=5
 pymc-sphinx-theme==0.1
-sphinx-design
+seaborn
```

Comment:
seaborn should not be added here. Sorry I missed that in the previous round.

These are the requirements to **build** the website (you'll see not even pymc is here). The requirements to write notebooks are the `-write` ones. We could add seaborn there, but I would argue against this. Barely anybody knows about this, they see the notebook on the website and try to run it, often not even finding this repo at all. As seaborn is not a pymc dependency, I think it should be added using https://www.pymc.io/projects/docs/en/latest/contributing/jupyter_style.html#extra-dependencies

Reply:
That's a good point! Seaborn has been removed from the `requirements-docs.txt` and added to the notebook-specific metadata.